skll/config.py

```diff
@@ -474,6 +474,26 @@ def _parse_config_file(config_path):
     # using the defaults?
     do_grid_search = config.getboolean("Tuning", "grid_search")
 
+    # Check `fixed_parameter_list`/`param_grid_list` if `do_grid_search`
+    # is True
+    if do_grid_search:
+        for learner, fixed_params, params in zip(fixed_parameter_list,
+                                                 param_grid_list,
+                                                 learners):
+            overlap_params = set(fixed_params).intersection(set(params))
+            if overlap_params:
+                logger.warning('Both "fixed_parameters" and "param_grids" were'
+                               'specified or the default values were used for '
+                               '{}, which resulted in conflicts between '
```

Comment:
As I understand it, specifying `grid_search` and then not specifying `param_grids` will default to the default parameter grids. So, let's say that someone wants to do grid search and doesn't specify the parameter grids. Then, they go to specify their fixed parameters and they don't really know what conflicts there may be between the defaults and the values they pick (probably rare, but still could happen). There could be a conflict between a parameter value they specified as fixed and a paramter grid value they did not specify at all. Does that make sense?